uml/sequence_diagrams/ClockView.txt

```diff
@@ -0,0 +1,46 @@
+@startuml
+participant DeactivatedClock 
+participant ClockView 
+participant BlueTooth
+participant Battery
+participant DisplayError 

```

Comment:
I see DisplayError on a few of your diagrams - should we account for it anywhere else (e.g. class diagrams)

Reply:
Maybe makes sense in state diagram if it's not too messy. But it's not necessary I think, this is sufficient I think, it won't add anything in other diagrams.